README.md

```diff
@@ -1,43 +1,58 @@
 # Instrument Addon Custom Device
 
-The **Instrument Addon Custom Device** allows users to define instrument commands generically. It supports: NI-VISA(GPIB, RS232, RS485, RS422), TCP, UDP, AK TCP, and AK RS232 as base protocols. The communication structure should be command-response. The instrument doesn't initiate any of the communications. Communication rates should be 10s of Hz, though the add-on supports decimation so the NI VeriStand PCL can still be fast.
+The **Instrument Addon Custom Device** allows users to generically define instrument commands.
 
-### Quick Start Documentation
+The add-on supports the following base protocols:
+- NI-VISA (GPIB, RS232, RS485, and RS422)
+- TCP
+- UDP
+- AK TCP
+- AK RS232
 
-- Built: %Public%\Documents\National Instruments\NI VeriStand (Year)\Custom Devices\Instrument Addon\Windows\Quick Start Documentation
-- Source: Source\Quick Start Documentation
+You should set the communication structure to *command-response*. The instrument does not initiate communications. Communication rates should fall between 10 and 19 Hertz. The add-on supports decimation for a fast VeriStand Primary Control Loop.
```

Comment:
The communication rate as it is suggested now (between 10 and 19 Hertz) is a bit misleading for the users, as the Instrument Addon can run lower than 10 Hz and higher than 19 Hz.

We cannot guarantee a successful communication for hundred or thousand of hertz, hence our suggestion to run at lower frequencies in the range of 10s (tens) of Hertz.

Reply:
Ah. Okay. Would it make sense to say the communication rate should be lower than 100 Hertz? Saying 'in the range of tens' is a little confusing (At least for me. does it make sense to those using this addon?)

Reply:
We did not do any specific benchmark, and it may work fine for over 100 Hz, but if it does not, we can't do much about it. So we should say it like it is a feature and not a limitation, without specifying any fixed value. 😅 

So we are counting on you to come up with something that says the Addon will work fine at low rates, and may not work well at higher ones. 😉

Reply:
The original wording (10s of Hz) came from the Application Engineer that first started the development on this custom device. When we made the custom device NI official we just left the wording as it was, as it made sense for me. But I am not a user, nor a Tech Writer, just a mere development guy. 😉